tests/RadForce.in

```diff
@@ -13,7 +13,7 @@ amr.v              = 0       # verbosity in Amr
 # *****************************************************************
 # Resolution and refinement
 # *****************************************************************
-amr.n_cell          = 128 4 4
+amr.n_cell          = 64 4 4
 amr.max_level       = 0     # number of levels = max_level + 1
 amr.blocking_factor = 4     # grid size must be divisible by this
```

Comment:
GPU performance can be improved if the grid parameters are set such that the whole simulation fits in a single box. This should be possible by setting `amr.max_grid_size_x = 128`. Then it may not be necessary to reduce the size of the problem.